resources/views/inc/theme_styles.blade.php

```diff
@@ -1,15 +1,9 @@
-@basset('https://unpkg.com/bootstrap@5.2.3/dist/css/bootstrap.min.css')
-@basset('https://unpkg.com/@coreui/coreui@4.2.4/dist/css/coreui.min.css')
-@basset('https://unpkg.com/animate.css@4.1.1/animate.compat.css')
-@basset('https://unpkg.com/noty@3.2.0-beta-deprecated/lib/noty.css')
```

Comment:
Note to self: noty and animate aren't needed here, because they're provided by CRUD in the `scripts` and `styles` files: https://github.com/Laravel-Backpack/CRUD/tree/v6/src/resources/views/ui/inc